### PR TITLE
Bring focus to main window when clicking preferences

### LINKED ISSUE #105: Bring focus to main window after clicking "Preferences..."
Currently, it simply stays in the background while doing that.

### PR DESCRIPTION
**Pull request by leo**
This closes #105

### CODE REVIEW

**Comment by leo** [comment]
@rauchg I would appreciate being a collaborator, btw 😊

File: menu.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ module.exports = function createMenu ({ createWindow, updatePlugins }) {
           click (item, focusedWindow) {
             if (focusedWindow) {
               focusedWindow.rpc.emit('preferences');
+              return;
+            } else if (!mainWindow.isFocused()) {
+              mainWindow.on('focus', () => mainWindow.rpc.emit('preferences'));
```
Comment (rauchg) [inline menu.js]: - What if there's no `mainWindow` (all closed)? 
- If you don't use `once`, wouldn't preferences fire every subsequent time the window gets `focus`?

**Comment by rauchg** [comment]
Needs rebase

**Comment by leo** [comment]
Figured out that it makes a lot more sense to spawn a new window and open the preferences there instead of putting one of the open windows back into focus (since there are multiple windows, we shouldn't just pick one).

So I've created https://github.com/zeit/hyperterm/pull/235, which does exactly that.

**Pull request closed without merging**